src/test/resources/pages/instructorFeedbackResultsAjaxByQuestionViewForHelperOne.html

```diff
@@ -1,684 +1,701 @@
+                              <ul style="list-style-type: disc;margin-left: 20px;">
+                                 <li>
+                                    FlexiCommand
+                                 </li>
+                                 <li>
+                                    PowerSearch
+                                 </li>
+                                 <li>
+                                    GoodUI
+                                 </li>
+                                 <li>
+                                    Google Integration
+                                 </li>
+                              </ul>
+                           </span>
+                        </span>
+                     </span>
+                  </div>
+               </div>
+               <div class="panel-collapse collapse" id="panelBodyCollapse-7">
+                  <div class="panel-body padding-0" id="questionBody-6">
+                     <div class="col-sm-12 no-response">
+                        <i class="text-muted">
+                           There are no responses for this question.
+                        </i>
+                     </div>
+                  </div>
                </div>
             </div>
-         </div>
-      </div>
-      <div class="panel panel-info">
-         <div class="panel-heading ajax_submit" data-target="#panelBodyCollapse-10" id="panelHeading-10" style="cursor: pointer;">
-            <form action="/page/instructorFeedbackResultsPage" class="seeMoreForm-10" id="seeMore-10" style="display:none;">
-               <input name="courseid" type="hidden" value="CFResultsUiT.CS2104"/>
-                              <input name="fsname" type="hidden" value="First Session"/>
-                              <input name="user" type="hidden" value="CFResultsUiT.helper1"/>
-                              <input name="frgroupbyteam" type="hidden" value="null"/>
-                              <input name="frsorttype" type="hidden" value="question"/>
-                              <input id="showStats-10" name="frshowstats" type="hidden" value="on"/>
-                              <input name="questionnum" type="hidden" value="10"/>
-                              <input name="frgroupbysection" type="hidden" value="All"/>
-                           </form>
-            <div class="display-icon pull-right">
-               <span class="glyphicon glyphicon-chevron-down pull-right">
-               </span>
+            <div class="panel panel-info">
+               <div class="panel-heading ajax_submit" data-target="#panelBodyCollapse-8" id="panelHeading-8" style="cursor: pointer;">
+                  <form action="/page/instructorFeedbackResultsPage" class="seeMoreForm-8" id="seeMore-8" style="display:none;">
+                     <input name="courseid" type="hidden" value="CFResultsUiT.CS2104"/>
+                                          <input name="fsname" type=
```

Comment:
Hmmm.... this is very weird. The same thing happens in the current master (it includes the html tags in the generated file). But when I try in AdminEmailPageUiTest, the generated file is ok